tests/openassetio/test/managerValidator/test_commandLine.py

```diff
@@ -0,0 +1,128 @@
+            manager_factory, logger, harness_factory, case_loader, harness):
+        # setup
+
+        argv = Mock()
+        parsed = command_line_parser.parse.return_value
+
+        # action
+
+        result = commandLine.execute(
+            argv, command_line_parser, a_module, harness_factory)
+
+        # confirm
+
+        command_line_parser.parse.assert_called_once_with(argv)
+        harness_factory.createHostInterface.assert_called_once_with()
+        harness_factory.createLogger.assert_called_once_with()
+        harness_factory.createManagerFactory.assert_called_once_with(logger)
+        harness_factory.createSessionWithManager.assert_called_once_with(
+            parsed.fixtures["identifier"], host_interface, logger, manager_factory)
+        harness_factory.createLoader.assert_called_once_with(parsed.fixtures, session)
+        harness_factory.createHarness.assert_called_once_with(unittest.main, case_loader)
+        harness.executeTests.assert_called_once_with(parsed.extraArgs, a_module)
+        assert result is harness.executeTests.return_value
+
+
+class Test_Parser_parse:
+    def test_loads_fixtures_and_returns_with_extra_args(self, fixture_loader):
+        parser = commandLine.Parser(fixture_loader)
+
+        parsed = parser.parse([
+            "managerValidator", "--fixtures", "/some/fixtures.json", "-v", "extra", "arg"])
+
+        fixture_loader.load.assert_called_once_with("/some/fixtures.json")
+        assert parsed.fixtures is fixture_loader.load.return_value
+        assert parsed.extraArgs == ["-v", "extra", "arg"]
+
+
+class Test_FixtureLoader_load:
+    def test_retrieves_dict_from_python_file(self, fixture_dict, fixture_file_path):
+        fixture_loader = commandLine.FixtureLoader()
+
+        actual_dict = fixture_loader.load(fixture_file_path)
+
+        assert fixture_dict == actual_dict
+
+
+@pytest.fixture
+def command_line_parser():
+    parser = mock.create_autospec(commandLine.Parser)
+    parsed = mock.create_autospec(commandLine.Parser.Parsed)
+    parser.parse.return_value = parsed
+    return parser
+
+
+@pytest.fixture
+def a_module():
+    return mock.create_autospec(types.ModuleType)
+
+
+@pytest.fixture
+def harness_factory(harness, case_loader, host_interface, session, manager_factory, logger):
+    factory = mock.create_autospec(
+        validatorHarness.ValidatorHarnessFactory, instance=True, spec_set=True)
+    factory.createHarness.return_value = harness
+    factory.createLoader.return_value = case_loader
+    factory.createHostInterface.return_value = host_interface
+    factory.createSessionWithManager.return_value = session
+    factory.createManagerFactory.return_value = manager_factory
+    factory.createLogger.return_value = logger
+    return factory
+
+
+@pytest.fixture
+def host_interface():
+    return mock.create_autospec(
+        validatorHarness.ValidatorHarnessHostInterface, instance=True, spec_set=True)
+
+
+@pytest.fixture
```

Comment:
[fix] Existing tests prefix fixtures that provide mocks with `mock_` to make it clear it is not testing a real system component. Eg: the ambiguity between `L57` and the fixture of the same name.

Reply:
I like it - `a_` for "real" things and `mock_` for mocks(?) - done in 03c84f9